go/obscuronode/enclave/crypto.go

```diff
@@ -30,16 +30,16 @@ func EncryptTx(tx L2Tx) common2.EncryptedTx {
 	return bytes
 }
 
-func encryptTransactions(transactions Transactions) common2.EncryptedTransactions {
+func EncryptTransactions(transactions Transactions) common2.EncryptedTransactions {
```

Comment:
why does this need to be public now?

Reply:
It needed to be public when I fixed the UT the first time, but since I moved the tests bc of cyclic dependencies, no need for this func to be public anymore. Nice one.